src/main/resources/application.yaml

```diff
@@ -6,6 +6,8 @@ spring:
     url: ${DATASOURCE_URL}
     username: ${DATASOURCE_USERNAME}
     password: ${DATASOURCE_PASSWORD}
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
Could you explain what does this property change?
If you want to use it in tests only, it'd be better to either set in `@SpringBootTest(properties = ...)` or create `application.yaml` in `test/main/resources` and set it there.

Reply:
we override `taskFetchingService` in  `ServiceTestConfiguration` and `SpringBootTest` doesn't like it without special option. Anyway, I realized that I do not need config, so it should be removed.

Reply:
Interesting issue - with removing `ServiceTestConfiguration`  if I run all tests (run ChatStateServiceTest) - tests pass. But if I run one test (e.g. StartTest) - it fails. With `ServiceTestConfiguration` and `allow-bean-definition-overriding: true` - all tests pass. For me, tests should pass even without `ServiceTestConfiguration`, but no :(